Source/JavaScriptCore/wasm/WasmSlowPaths.cpp

```diff
@@ -191,11 +191,13 @@ static inline bool jitCompileSIMDFunction(Wasm::LLIntCallee* callee, Wasm::Insta
             tierUpCounter.m_compilationStatus = Wasm::LLIntTierUpCounter::CompilationStatus::Compiling;
             break;
         case Wasm::LLIntTierUpCounter::CompilationStatus::Compiling:
-            // This spinlock is bad, but this is only temporary.
+            // FIXME: This spinlock is bad, but this is only temporary. https://bugs.webkit.org/show_bug.cgi?id=274326
             continue;
         case Wasm::LLIntTierUpCounter::CompilationStatus::Compiled:
-            RELEASE_ASSERT(!!callee->replacement(instance->memory()->mode()));
-            return true;
+            if (!!callee->replacement(instance->memory()->mode()))
+                return true;
+            callee->clearOtherReplacementsIfExisting(instance->memory()->mode());
+            tierUpCounter.m_compilationStatus = Wasm::LLIntTierUpCounter::CompilationStatus::NotCompiled;
```

Comment:
Let's add a `continue` after this line to make it explicit.  It also ensures that we don't inadvertently fall thru if someone adds a case for a 4th status in the future.

Reply:
Changed.